README.md

```diff
@@ -32,3 +32,9 @@ Then you can run yarn to build or test the project:
 * `yarn test` - run the tests
 * `yarn watch-test` - continuously run the tests
 * `yarn cli -- <arguments>` - run the CLI test tool (try `--help` for options)
+
+# Debugging Tests
+
+On MAC OS X with WebStorm v2017.2 debugging tests may not work.
+See https://github.com/facebook/jest/issues/1652 for more details.
+The described solution there is to add `--env jest-environment-node-debug` as argument to the runtime configuration.
```

Comment:
As it only affects me for now, I'd say lets keep it the way it is and make it more comfortable as soon as more people having the problem, as it is probably fixed at some point in the future.